store/leveldb/options.go

```diff
@@ -69,6 +69,10 @@ func (o *Options) SetMaxOpenFiles(n int) {
 	C.leveldb_options_set_max_open_files(o.Opt, C.int(n))
 }
 
+func (o *Options) SetMaxFileSize(n int) {
```

Comment:
does the new LevelDB have a function to set manifest file size like RocksDB does?

Reply:
Yes it does - it's a recent addition.

Reply:
can we support this? Or SetMaxFileSize already does?

Reply:
We already support it - it's `MaxFileSize` in `LevelDBConfig` - you can modify it by changing Ledis config file. The default is the exact same value you've set in your fork of LevelDB for Ledis 0.5